doc/tor.md

```diff
@@ -211,3 +209,105 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+
+- For maximum privacy, it is preferable to disable accepting incoming connections.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks) for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  The `onlynet=onion` configuration option can potentially ensure the node only attempts to connect over
+  the Tor network. The decision to use this option should consider the following trade-offs:
+
+  Only connecting over Tor is more private when you combine it with no reachable IPv4/IPv6 address,
+  in particular if you want to broadcast transactions without them being correlatable with your IP.
+
+  **Sybil Attacks**
+
+  On the other hand, if you only make random Tor connections, you're much more vulnerable to Sybil attacks.
+  As Tor addresses may be created at no cost, an attacker can potentially flood the network with many Tor
+  nodes and receive all of the outbound Tor connections an `onlynet=tor` node makes.
+
+  This is significantly less a concern with IPv4/IPv6 (especially with asmap) due to cost of getting IPs in many
+  networks. It's also less a concern if you have `-addnode` connections to trusted peers (even if they're onion addresses).
+
+  If all of your connections are controlled by a Sybil attacker, they can easily prevent you from seeing confirmed transactions
+  and, with more difficulty, even trick your node into falsely reporting a transaction as confirmed on the blockchain with most
+  cumulative “chainwork”.
+
+  **Network Partitioning**
+
+  If too many nodes use `onlynet=onion`, it could become difficult for onion nodes to communicate with clearnet nodes,
+  preventing the Tor network from seeing recent transactions and blocks. It is essential that some nodes access both
+  clearnet and Tor, or use `onlynet=i2p` with `onlynet=onion`.
+
+## Setup Guide: Debian/Ubuntu
+
+Assuming you already have an installation of Bitcoin Core:
+
+**1. Install Tor:**
+
+To install Tor, run:
+
+```
+sudo apt install tor
+```
+
+**2. Edit Tor configuration file:**
+
+Open the `torrc` file with your text editor. This example uses nano:
+
+```
+sudo nano /etc/tor/torrc
+```
+Ensure that `torrc` has these settings, and save:
+
+```
+ControlPort 9051
+CookieAuthentication 1
+CookieAuthFileGroupReadable 1
+```
+
+**3. Add user to Tor group ("satoshi" in this example):**
+
+```
+sudo usermod -a -G debian-tor satoshi
+```
+
+**4. Restart System**
+
+```
+sudo reboot
```

Comment:
I think you can drop the `sudo` instructions in this section. They are context-dependent.

Reply:
As mentioned in this answer: https://askubuntu.com/a/1006034, `sudo` is mentioned in most of the tutorials for compatibility reasons and these steps in the example are included for all newbies who are trying Bitcoin Core with Tor for the first time on Ubuntu or other Debian based Linux distributions.